.changeset/tasty-tomatoes-turn.md

```diff
@@ -0,0 +1,17 @@
+---
+'openzeppelin-solidity': major
+---
+
+Replaces `toString(int256)` with `toStringSigned(int256)`
+
+# WHAT the breaking change is?
+
+- The toString(int256) is now replaced with toStringSigned(int256)
+
+# WHY the change was made?
+
+- The change was made as invoking `toString(int256)` with an integer literal from another contract was failing with the error `Member "toString" not unique` since `toString(1)` qualifies for both int256 & uint256, hence replaced `toString(int256)` with `toStringSigned(int256)`.
+
+# HOW a consumer should update their code?
+
+- Replace the instances of `toString(int256)` with `toStringSigned(int256)`.
```

Comment:
Changeset entry should be one liner. For reference, here is how it should end up looking like in the Changelog:

![Capture d’écran du 2023-06-16 18-46-06](https://github.com/OpenZeppelin/openzeppelin-contracts/assets/2432299/3c5f488d-cfac-4b97-8d95-ee17b7c41e60)

```suggestion
`Strings`: Rename `toString(int256)` to `toStringSigned(int256)`.
```